In [1]:
import mteb
from sentence_transformers import SentenceTransformer
import os


In [2]:
os.environ['CUDA_VISIBLE_DEVICES']="1"

In [4]:
#model_name = "unb-labia/BERTomelo-ModernBERT-Base-8k-Experimental"
#model_name = "Tropic-AI/moBERTo"
#model_name = "Qwen/Qwen3-Embedding-0.6B"
#model_name = "unb-labia/BERTomelo-ModernBERT-Base-v1"
#model_name = "neuralmind/bert-base-portuguese-cased"
model_name = "unb-labia/BERTomelo-ModernBERT-Large-STS-8k"

model = mteb.get_model(model_name, device='cuda:1')

Can't get file config_sentence_transformers.json of unb-labia/BERTomelo-ModernBERT-Large-STS-8k: 404 Client Error. (Request ID: Root=1-6a85c0ab-795f55bc73835f04547dc7d5;11e7ec17-b015-4891-92cc-5f5ef6613a8e)

Entry Not Found for url: https://huggingface.co/unb-labia/BERTomelo-ModernBERT-Large-STS-8k/resolve/main/config_sentence_transformers.json.
Can't get file sentence_bert_config.json of unb-labia/BERTomelo-ModernBERT-Large-STS-8k: 404 Client Error. (Request ID: Root=1-6a85c0ac-7d8e13bb4bdd61b01d2a766e;44e58465-20b6-4651-a187-a40c6fbf1115)

Entry Not Found for url: https://huggingface.co/unb-labia/BERTomelo-ModernBERT-Large-STS-8k/resolve/main/sentence_bert_config.json.
Can't get file modules.json of unb-labia/BERTomelo-ModernBERT-Large-STS-8k: 404 Client Error. (Request ID: Root=1-6a85c0ac-1db6c6eb5bbdad1a133eb9fa;024befe7-662c-4caa-a5ab-7bffa2e6a652)

Entry Not Found for url: https://huggingface.co/unb-labia/BERTomelo-ModernBERT-Large-STS-8k/resolve/main/modules.json.


README.md:   0%|          | 0.00/85.0 [00:00<?, ?B/s]

Can't get file sentence_bert_config.json of unb-labia/BERTomelo-ModernBERT-Large-STS-8k: 404 Client Error. (Request ID: Root=1-6a85c0b0-6cb3c86866f4b214530c9e1c;cba318a6-2c18-4650-a767-5488f995dfd8)

Entry Not Found for url: https://huggingface.co/unb-labia/BERTomelo-ModernBERT-Large-STS-8k/resolve/0fc6ecde6e8d9c81d687c1e89d51f70749739a51/sentence_bert_config.json.
Can't get file config_sentence_transformers.json of unb-labia/BERTomelo-ModernBERT-Large-STS-8k: 404 Client Error. (Request ID: Root=1-6a85c0b0-0ed94db5321626243b92dd8b;90c84429-23e8-4002-ba1a-a84da1661c3f)

Entry Not Found for url: https://huggingface.co/unb-labia/BERTomelo-ModernBERT-Large-STS-8k/resolve/0fc6ecde6e8d9c81d687c1e89d51f70749739a51/config_sentence_transformers.json.


model.safetensors: reconstructing file:   0%|          |  0.00B / 1.51GB            

model.safetensors: downloading bytes:           |  0.00B            

Loading weights:   0%|          | 0/174 [00:00<?, ?it/s]

tokenizer_config.json:   0%|          | 0.00/413 [00:00<?, ?B/s]

tokenizer.json:   0%|          | 0.00/2.11M [00:00<?, ?B/s]

In [5]:
tasks = mteb.get_tasks(tasks=["MultiLongDocReranking"], languages=["por"])
tasks


MTEBTasks(MultiLongDocReranking(name='MultiLongDocReranking', languages=['por']),)

In [6]:
results = mteb.evaluate(model, tasks=tasks, encode_kwargs={'batch_size': 1, 'show_progress_bar': True})

Evaluating tasks:   0%|          | 0/1 [00:00<?, ?it/s]

Flattening the indices:   0%|          | 0/1600 [00:00<?, ? examples/s]

Standardizing text corpus format:   0%|          | 0/1600 [00:00<?, ? examples/s]

[W819 11:44:14.197026000 CUDACachingAllocator.cpp:3933] memory allocation failed with OOM on device 1 while trying to allocate 1073741824 bytes (free: 472907776, total: 12884377600).
[W819 11:44:14.255473059 CUDACachingAllocator.cpp:3933] memory allocation failed with OOM on device 1 while trying to allocate 1073741824 bytes (free: 475004928, total: 12884377600).


OutOfMemoryError: CUDA out of memory. Tried to allocate 1024.00 MiB. GPU 1 has a total capacity of 12.00 GiB of which 453.00 MiB is free. Including non-PyTorch memory, this process has 17179869184.00 GiB memory in use. Of the allocated memory 9.43 GiB is allocated by PyTorch, and 1.09 GiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://docs.pytorch.org/docs/stable/notes/cuda.html#optimizing-memory-usage-with-pytorch-cuda-alloc-conf)

In [8]:
print(results.task_results[0].main_score)
results.task_results[0].scores

0.90925


{'test': [{'ndcg_at_1': 0.81,
   'ndcg_at_3': 0.89113,
   'ndcg_at_5': 0.89909,
   'ndcg_at_10': 0.90925,
   'ndcg_at_20': 0.90925,
   'ndcg_at_100': 0.90925,
   'ndcg_at_1000': 0.90925,
   'map_at_1': 0.81,
   'map_at_3': 0.87083,
   'map_at_5': 0.87508,
   'map_at_10': 0.87955,
   'map_at_20': 0.87955,
   'map_at_100': 0.87955,
   'map_at_1000': 0.87955,
   'recall_at_1': 0.81,
   'recall_at_3': 0.95,
   'recall_at_5': 0.97,
   'recall_at_10': 1.0,
   'recall_at_20': 1.0,
   'recall_at_100': 1.0,
   'recall_at_1000': 1.0,
   'accuracy': 0.81,
   'precision_at_1': 0.81,
   'precision_at_3': 0.31667,
   'precision_at_5': 0.194,
   'precision_at_10': 0.1,
   'precision_at_20': 0.05,
   'precision_at_100': 0.01,
   'precision_at_1000': 0.001,
   'mrr_at_1': 0.81,
   'mrr_at_3': 0.8708333333333332,
   'mrr_at_5': 0.8750833333333334,
   'mrr_at_10': 0.8795476190476191,
   'mrr_at_20': 0.8795476190476191,
   'mrr_at_100': 0.8795476190476191,
   'mrr_at_1000': 0.8795476190476191,
   'nauc_nd

# BERTimbau
- main: 0.90925

{'test': [{'ndcg_at_1': 0.81,
   'ndcg_at_3': 0.89113,
   'ndcg_at_5': 0.89909,
   'ndcg_at_10': 0.90925,
   'ndcg_at_20': 0.90925,
   'ndcg_at_100': 0.90925,
   'ndcg_at_1000': 0.90925,
   'map_at_1': 0.81,
   'map_at_3': 0.87083,
   'map_at_5': 0.87508,
   'map_at_10': 0.87955,
   'map_at_20': 0.87955,
   'map_at_100': 0.87955,
   'map_at_1000': 0.87955,
   'recall_at_1': 0.81,
   'recall_at_3': 0.95,
   'recall_at_5': 0.97,
   'recall_at_10': 1.0,
   'recall_at_20': 1.0,
   'recall_at_100': 1.0,
   'recall_at_1000': 1.0,
   'accuracy': 0.81,
   'precision_at_1': 0.81,
   'precision_at_3': 0.31667,
   'precision_at_5': 0.194,
   'precision_at_10': 0.1,
   'precision_at_20': 0.05,
   'precision_at_100': 0.01,
   'precision_at_1000': 0.001,
   'mrr_at_1': 0.81,
   'mrr_at_3': 0.8708333333333332,
   'mrr_at_5': 0.8750833333333334,
   'mrr_at_10': 0.8795476190476191,
   'mrr_at_20': 0.8795476190476191,
   'mrr_at_100': 0.8795476190476191,
   'mrr_at_1000': 0.8795476190476191,
   'nauc_ndcg_at_1_max': np.float64(0.5068625910210074),
   'nauc_ndcg_at_1_std': np.float64(0.5462962962962966),
   'nauc_ndcg_at_1_diff1': np.float64(0.758604431871759),
   'nauc_ndcg_at_3_max': np.float64(0.49455446180249907),
   'nauc_ndcg_at_3_std': np.float64(0.6259845509079761),
   'nauc_ndcg_at_3_diff1': np.float64(0.7542927565831729),
   'nauc_ndcg_at_5_max': np.float64(0.5298887452460461),
   'nauc_ndcg_at_5_std': np.float64(0.6124803859472983),
   'nauc_ndcg_at_5_diff1': np.float64(0.7544103264148956),
   'nauc_ndcg_at_10_max': np.float64(0.5067920160920631),
   'nauc_ndcg_at_10_std': np.float64(0.5919905079723983),
   'nauc_ndcg_at_10_diff1': np.float64(0.7508611991903524),
   'nauc_ndcg_at_20_max': np.float64(0.5067920160920631),
   'nauc_ndcg_at_20_std': np.float64(0.5919905079723983),
   'nauc_ndcg_at_20_diff1': np.float64(0.7508611991903524),
   'nauc_ndcg_at_100_max': np.float64(0.5067920160920631),
   'nauc_ndcg_at_100_std': np.float64(0.5919905079723983),
   'nauc_ndcg_at_100_diff1': np.float64(0.7508611991903524),
   'nauc_ndcg_at_1000_max': np.float64(0.5067920160920631),
   'nauc_ndcg_at_1000_std': np.float64(0.5919905079723983),
   'nauc_ndcg_at_1000_diff1': np.float64(0.7508611991903524),
   'nauc_map_at_1_max': np.float64(0.5068625910210074),
   'nauc_map_at_1_std': np.float64(0.5462962962962966),
   'nauc_map_at_1_diff1': np.float64(0.758604431871759),
   'nauc_map_at_3_max': np.float64(0.49851236537064486),
   'nauc_map_at_3_std': np.float64(0.6017276775844096),
   'nauc_map_at_3_diff1': np.float64(0.7521082720065695),
   'nauc_map_at_5_max': np.float64(0.513631643995239),
   'nauc_map_at_5_std': np.float64(0.5950858692160387),
   'nauc_map_at_5_diff1': np.float64(0.7521163887887503),
   'nauc_map_at_10_max': np.float64(0.5060449854322147),
   'nauc_map_at_10_std': np.float64(0.5881643365913135),
   'nauc_map_at_10_diff1': np.float64(0.7509668237875694),
   'nauc_map_at_20_max': np.float64(0.5060449854322147),
   'nauc_map_at_20_std': np.float64(0.5881643365913135),
   'nauc_map_at_20_diff1': np.float64(0.7509668237875694),
   'nauc_map_at_100_max': np.float64(0.5060449854322147),
   'nauc_map_at_100_std': np.float64(0.5881643365913135),
   'nauc_map_at_100_diff1': np.float64(0.7509668237875694),
   'nauc_map_at_1000_max': np.float64(0.5060449854322147),
   'nauc_map_at_1000_std': np.float64(0.5881643365913135),
   'nauc_map_at_1000_diff1': np.float64(0.7509668237875694),
   'nauc_recall_at_1_max': np.float64(0.5068625910210074),
   'nauc_recall_at_1_std': np.float64(0.5462962962962966),
   'nauc_recall_at_1_diff1': np.float64(0.758604431871759),
   'nauc_recall_at_3_max': np.float64(0.4667600373482738),
   'nauc_recall_at_3_std': np.float64(0.7948179271708677),
   'nauc_recall_at_3_diff1': np.float64(0.7735760971055076),
   'nauc_recall_at_5_max': np.float64(0.7394179894179912),
   'nauc_recall_at_5_std': np.float64(0.7969187675070006),
   'nauc_recall_at_5_diff1': np.float64(0.7868036103330251),
   'nauc_recall_at_10_max': nan,
   'nauc_recall_at_10_std': nan,
   'nauc_recall_at_10_diff1': nan,
   'nauc_recall_at_20_max': nan,
   'nauc_recall_at_20_std': nan,
   'nauc_recall_at_20_diff1': nan,
   'nauc_recall_at_100_max': nan,
   'nauc_recall_at_100_std': nan,
   'nauc_recall_at_100_diff1': nan,
   'nauc_recall_at_1000_max': nan,
   'nauc_recall_at_1000_std': nan,
   'nauc_recall_at_1000_diff1': nan,
   'nauc_precision_at_1_max': np.float64(0.5068625910210074),
   'nauc_precision_at_1_std': np.float64(0.5462962962962966),
   'nauc_precision_at_1_diff1': np.float64(0.758604431871759),
   'nauc_precision_at_3_max': np.float64(0.46676003734827193),
   'nauc_precision_at_3_std': np.float64(0.7948179271708666),
   'nauc_precision_at_3_diff1': np.float64(0.7735760971055091),
   'nauc_precision_at_5_max': np.float64(0.7394179894179901),
   'nauc_precision_at_5_std': np.float64(0.796918767507004),
   'nauc_precision_at_5_diff1': np.float64(0.7868036103330176),
   'nauc_precision_at_10_max': np.float64(1.0),
   'nauc_precision_at_10_std': np.float64(1.0),
   'nauc_precision_at_10_diff1': np.float64(1.0),
   'nauc_precision_at_20_max': np.float64(1.0),
   'nauc_precision_at_20_std': np.float64(1.0),
   'nauc_precision_at_20_diff1': np.float64(1.0),
   'nauc_precision_at_100_max': np.float64(1.0),
   'nauc_precision_at_100_std': np.float64(1.0),
   'nauc_precision_at_100_diff1': np.float64(1.0),
   'nauc_precision_at_1000_max': np.float64(1.0),
   'nauc_precision_at_1000_std': np.float64(1.0),
   'nauc_precision_at_1000_diff1': np.float64(1.0),
   'nauc_mrr_at_1_max': np.float64(0.5068625910210074),
   'nauc_mrr_at_1_std': np.float64(0.5462962962962966),
   'nauc_mrr_at_1_diff1': np.float64(0.758604431871759),
   'nauc_mrr_at_3_max': np.float64(0.49851236537064486),
   'nauc_mrr_at_3_std': np.float64(0.6017276775844096),
   'nauc_mrr_at_3_diff1': np.float64(0.7521082720065695),
   'nauc_mrr_at_5_max': np.float64(0.513631643995239),
   'nauc_mrr_at_5_std': np.float64(0.5950858692160387),
   'nauc_mrr_at_5_diff1': np.float64(0.7521163887887503),
   'nauc_mrr_at_10_max': np.float64(0.5060449854322147),
   'nauc_mrr_at_10_std': np.float64(0.5881643365913135),
   'nauc_mrr_at_10_diff1': np.float64(0.7509668237875694),
   'nauc_mrr_at_20_max': np.float64(0.5060449854322147),
   'nauc_mrr_at_20_std': np.float64(0.5881643365913135),
   'nauc_mrr_at_20_diff1': np.float64(0.7509668237875694),
   'nauc_mrr_at_100_max': np.float64(0.5060449854322147),
   'nauc_mrr_at_100_std': np.float64(0.5881643365913135),
   'nauc_mrr_at_100_diff1': np.float64(0.7509668237875694),
   'nauc_mrr_at_1000_max': np.float64(0.5060449854322147),
   'nauc_mrr_at_1000_std': np.float64(0.5881643365913135),
   'nauc_mrr_at_1000_diff1': np.float64(0.7509668237875694),
   'hit_rate_at_1': 0.81,
   'hit_rate_at_3': 0.95,
   'hit_rate_at_5': 0.97,
   'hit_rate_at_10': 1.0,
   'hit_rate_at_20': 1.0,
   'hit_rate_at_100': 1.0,
   'hit_rate_at_1000': 1.0,
   'main_score': 0.90925,
   'hf_subset': 'pt',
   'languages': ['por-Latn'],
   'mteb_version': '2.19.5'}]}

# BERTOMelo Base 1k
- Main: 0.90153

{'test': [{'ndcg_at_1': 0.79,
   'ndcg_at_3': 0.87625,
   'ndcg_at_5': 0.89304,
   'ndcg_at_10': 0.90153,
   'ndcg_at_20': 0.90153,
   'ndcg_at_100': 0.90153,
   'ndcg_at_1000': 0.90153,
   'map_at_1': 0.79,
   'map_at_3': 0.85583,
   'map_at_5': 0.86533,
   'map_at_10': 0.86908,
   'map_at_20': 0.86908,
   'map_at_100': 0.86908,
   'map_at_1000': 0.86908,
   'recall_at_1': 0.79,
   'recall_at_3': 0.935,
   'recall_at_5': 0.975,
   'recall_at_10': 1.0,
   'recall_at_20': 1.0,
   'recall_at_100': 1.0,
   'recall_at_1000': 1.0,
   'accuracy': 0.79,
   'precision_at_1': 0.79,
   'precision_at_3': 0.31167,
   'precision_at_5': 0.195,
   'precision_at_10': 0.1,
   'precision_at_20': 0.05,
   'precision_at_100': 0.01,
   'precision_at_1000': 0.001,
   'mrr_at_1': 0.79,
   'mrr_at_3': 0.8558333333333333,
   'mrr_at_5': 0.8653333333333333,
   'mrr_at_10': 0.8690833333333333,
   'mrr_at_20': 0.8690833333333333,
   'mrr_at_100': 0.8690833333333333,
   'mrr_at_1000': 0.8690833333333333,
   'nauc_ndcg_at_1_max': np.float64(0.3860942121335997),
   'nauc_ndcg_at_1_std': np.float64(0.39643398177314954),
   'nauc_ndcg_at_1_diff1': np.float64(0.7878543775699133),
   'nauc_ndcg_at_3_max': np.float64(0.41001489003378083),
   'nauc_ndcg_at_3_std': np.float64(0.5091704526664773),
   'nauc_ndcg_at_3_diff1': np.float64(0.7759521723230424),
   'nauc_ndcg_at_5_max': np.float64(0.41684359543976957),
   'nauc_ndcg_at_5_std': np.float64(0.4726878872928627),
   'nauc_ndcg_at_5_diff1': np.float64(0.7917054295137484),
   'nauc_ndcg_at_10_max': np.float64(0.4068409670043563),
   'nauc_ndcg_at_10_std': np.float64(0.45148865269529376),
   'nauc_ndcg_at_10_diff1': np.float64(0.7773206147373102),
   'nauc_ndcg_at_20_max': np.float64(0.4068409670043563),
   'nauc_ndcg_at_20_std': np.float64(0.45148865269529376),
   'nauc_ndcg_at_20_diff1': np.float64(0.7773206147373102),
   'nauc_ndcg_at_100_max': np.float64(0.4068409670043563),
   'nauc_ndcg_at_100_std': np.float64(0.45148865269529376),
   'nauc_ndcg_at_100_diff1': np.float64(0.7773206147373102),
   'nauc_ndcg_at_1000_max': np.float64(0.4068409670043563),
   'nauc_ndcg_at_1000_std': np.float64(0.45148865269529376),
   'nauc_ndcg_at_1000_diff1': np.float64(0.7773206147373102),
   'nauc_map_at_1_max': np.float64(0.3860942121335997),
   'nauc_map_at_1_std': np.float64(0.39643398177314954),
   'nauc_map_at_1_diff1': np.float64(0.7878543775699133),
   'nauc_map_at_3_max': np.float64(0.4066575978234773),
   'nauc_map_at_3_std': np.float64(0.4745596085917763),
   'nauc_map_at_3_diff1': np.float64(0.7765435133003353),
   'nauc_map_at_5_max': np.float64(0.40876388491206755),
   'nauc_map_at_5_std': np.float64(0.45431819382778016),
   'nauc_map_at_5_diff1': np.float64(0.7826749354525935),
   'nauc_map_at_10_max': np.float64(0.40547546335665746),
   'nauc_map_at_10_std': np.float64(0.447258611617491),
   'nauc_map_at_10_diff1': np.float64(0.7775941862519323),
   'nauc_map_at_20_max': np.float64(0.40547546335665746),
   'nauc_map_at_20_std': np.float64(0.447258611617491),
   'nauc_map_at_20_diff1': np.float64(0.7775941862519323),
   'nauc_map_at_100_max': np.float64(0.40547546335665746),
   'nauc_map_at_100_std': np.float64(0.447258611617491),
   'nauc_map_at_100_diff1': np.float64(0.7775941862519323),
   'nauc_map_at_1000_max': np.float64(0.40547546335665746),
   'nauc_map_at_1000_std': np.float64(0.447258611617491),
   'nauc_map_at_1000_diff1': np.float64(0.7775941862519323),
   'nauc_recall_at_1_max': np.float64(0.3860942121335997),
   'nauc_recall_at_1_std': np.float64(0.39643398177314954),
   'nauc_recall_at_1_diff1': np.float64(0.7878543775699133),
   'nauc_recall_at_3_max': np.float64(0.42609351432880654),
   'nauc_recall_at_3_std': np.float64(0.714752567693745),
   'nauc_recall_at_3_diff1': np.float64(0.7754075989370115),
   'nauc_recall_at_5_max': np.float64(0.5327731092436959),
   'nauc_recall_at_5_std': np.float64(0.7170868347338918),
   'nauc_recall_at_5_diff1': np.float64(0.9477124183006561),
   'nauc_recall_at_10_max': nan,
   'nauc_recall_at_10_std': nan,
   'nauc_recall_at_10_diff1': nan,
   'nauc_recall_at_20_max': nan,
   'nauc_recall_at_20_std': nan,
   'nauc_recall_at_20_diff1': nan,
   'nauc_recall_at_100_max': nan,
   'nauc_recall_at_100_std': nan,
   'nauc_recall_at_100_diff1': nan,
   'nauc_recall_at_1000_max': nan,
   'nauc_recall_at_1000_std': nan,
   'nauc_recall_at_1000_diff1': nan,
   'nauc_precision_at_1_max': np.float64(0.3860942121335997),
   'nauc_precision_at_1_std': np.float64(0.39643398177314954),
   'nauc_precision_at_1_diff1': np.float64(0.7878543775699133),
   'nauc_precision_at_3_max': np.float64(0.42609351432880815),
   'nauc_precision_at_3_std': np.float64(0.7147525676937442),
   'nauc_precision_at_3_diff1': np.float64(0.7754075989370097),
   'nauc_precision_at_5_max': np.float64(0.5327731092436919),
   'nauc_precision_at_5_std': np.float64(0.7170868347338961),
   'nauc_precision_at_5_diff1': np.float64(0.9477124183006566),
   'nauc_precision_at_10_max': np.float64(1.0),
   'nauc_precision_at_10_std': np.float64(1.0),
   'nauc_precision_at_10_diff1': np.float64(1.0),
   'nauc_precision_at_20_max': np.float64(1.0),
   'nauc_precision_at_20_std': np.float64(1.0),
   'nauc_precision_at_20_diff1': np.float64(1.0),
   'nauc_precision_at_100_max': np.float64(1.0),
   'nauc_precision_at_100_std': np.float64(1.0),
   'nauc_precision_at_100_diff1': np.float64(1.0),
   'nauc_precision_at_1000_max': np.float64(1.0),
   'nauc_precision_at_1000_std': np.float64(1.0),
   'nauc_precision_at_1000_diff1': np.float64(1.0),
   'nauc_mrr_at_1_max': np.float64(0.3860942121335997),
   'nauc_mrr_at_1_std': np.float64(0.39643398177314954),
   'nauc_mrr_at_1_diff1': np.float64(0.7878543775699133),
   'nauc_mrr_at_3_max': np.float64(0.4066575978234773),
   'nauc_mrr_at_3_std': np.float64(0.4745596085917763),
   'nauc_mrr_at_3_diff1': np.float64(0.7765435133003353),
   'nauc_mrr_at_5_max': np.float64(0.40876388491206755),
   'nauc_mrr_at_5_std': np.float64(0.45431819382778016),
   'nauc_mrr_at_5_diff1': np.float64(0.7826749354525935),
   'nauc_mrr_at_10_max': np.float64(0.40547546335665746),
   'nauc_mrr_at_10_std': np.float64(0.447258611617491),
   'nauc_mrr_at_10_diff1': np.float64(0.7775941862519323),
   'nauc_mrr_at_20_max': np.float64(0.40547546335665746),
   'nauc_mrr_at_20_std': np.float64(0.447258611617491),
   'nauc_mrr_at_20_diff1': np.float64(0.7775941862519323),
   'nauc_mrr_at_100_max': np.float64(0.40547546335665746),
   'nauc_mrr_at_100_std': np.float64(0.447258611617491),
   'nauc_mrr_at_100_diff1': np.float64(0.7775941862519323),
   'nauc_mrr_at_1000_max': np.float64(0.40547546335665746),
   'nauc_mrr_at_1000_std': np.float64(0.447258611617491),
   'nauc_mrr_at_1000_diff1': np.float64(0.7775941862519323),
   'hit_rate_at_1': 0.79,
   'hit_rate_at_3': 0.935,
   'hit_rate_at_5': 0.975,
   'hit_rate_at_10': 1.0,
   'hit_rate_at_20': 1.0,
   'hit_rate_at_100': 1.0,
   'hit_rate_at_1000': 1.0,
   'main_score': 0.90153,
   'hf_subset': 'pt',
   'languages': ['por-Latn'],
   'mteb_version': '2.19.5'}]}

# Qwen/Qwen3-Embedding-0.6B
https://mteb-leaderboard.hf.space/tasks/MultiLongDocReranking?s.scores=subset%3Apt&d.scores=desc
- main: 0.9889

{'test': [{'ndcg_at_1': 0.975,
   'ndcg_at_3': 0.98696,
   'ndcg_at_5': 0.9889,
   'ndcg_at_10': 0.9889,
   'ndcg_at_20': 0.9889,
   'ndcg_at_100': 0.9889,
   'ndcg_at_1000': 0.9889,
   'map_at_1': 0.975,
   'map_at_3': 0.98417,
   'map_at_5': 0.98517,
   'map_at_10': 0.98517,
   'map_at_20': 0.98517,
   'map_at_100': 0.98517,
   'map_at_1000': 0.98517,
   'recall_at_1': 0.975,
   'recall_at_3': 0.995,
   'recall_at_5': 1.0,
   'recall_at_10': 1.0,
   'recall_at_20': 1.0,
   'recall_at_100': 1.0,
   'recall_at_1000': 1.0,
   'accuracy': 0.975,
   'precision_at_1': 0.975,
   'precision_at_3': 0.33167,
   'precision_at_5': 0.2,
   'precision_at_10': 0.1,
   'precision_at_20': 0.05,
   'precision_at_100': 0.01,
   'precision_at_1000': 0.001,
   'mrr_at_1': 0.975,
   'mrr_at_3': 0.9841666666666667,
   'mrr_at_5': 0.9851666666666666,
   'mrr_at_10': 0.9851666666666666,
   'mrr_at_20': 0.9851666666666666,
   'mrr_at_100': 0.9851666666666666,
   'mrr_at_1000': 0.9851666666666666,
   'nauc_ndcg_at_1_max': np.float64(0.8921568627450974),
   'nauc_ndcg_at_1_std': np.float64(0.8552754435107388),
   'nauc_ndcg_at_1_diff1': np.float64(1.0),
   'nauc_ndcg_at_3_max': np.float64(0.8920367504010691),
   'nauc_ndcg_at_3_std': np.float64(0.8975657645872014),
   'nauc_ndcg_at_3_diff1': np.float64(1.0),
   'nauc_ndcg_at_5_max': np.float64(0.896001475230866),
   'nauc_ndcg_at_5_std': np.float64(0.879718654914311),
   'nauc_ndcg_at_5_diff1': np.float64(1.0),
   'nauc_ndcg_at_10_max': np.float64(0.896001475230866),
   'nauc_ndcg_at_10_std': np.float64(0.879718654914311),
   'nauc_ndcg_at_10_diff1': np.float64(1.0),
   'nauc_ndcg_at_20_max': np.float64(0.896001475230866),
   'nauc_ndcg_at_20_std': np.float64(0.879718654914311),
   'nauc_ndcg_at_20_diff1': np.float64(1.0),
   'nauc_ndcg_at_100_max': np.float64(0.896001475230866),
   'nauc_ndcg_at_100_std': np.float64(0.879718654914311),
   'nauc_ndcg_at_100_diff1': np.float64(1.0),
   'nauc_ndcg_at_1000_max': np.float64(0.896001475230866),
   'nauc_ndcg_at_1000_std': np.float64(0.879718654914311),
   'nauc_ndcg_at_1000_diff1': np.float64(1.0),
   'nauc_map_at_1_max': np.float64(0.8921568627450974),
   'nauc_map_at_1_std': np.float64(0.8552754435107388),
   'nauc_map_at_1_diff1': np.float64(1.0),
   'nauc_map_at_3_max': np.float64(0.8942208462332385),
   'nauc_map_at_3_std': np.float64(0.8857437711926923),
   'nauc_map_at_3_diff1': np.float64(1.0),
   'nauc_map_at_5_max': np.float64(0.8959021810971549),
   'nauc_map_at_5_std': np.float64(0.8780411040820794),
   'nauc_map_at_5_diff1': np.float64(1.0),
   'nauc_map_at_10_max': np.float64(0.8959021810971549),
   'nauc_map_at_10_std': np.float64(0.8780411040820794),
   'nauc_map_at_10_diff1': np.float64(1.0),
   'nauc_map_at_20_max': np.float64(0.8959021810971549),
   'nauc_map_at_20_std': np.float64(0.8780411040820794),
   'nauc_map_at_20_diff1': np.float64(1.0),
   'nauc_map_at_100_max': np.float64(0.8959021810971549),
   'nauc_map_at_100_std': np.float64(0.8780411040820794),
   'nauc_map_at_100_diff1': np.float64(1.0),
   'nauc_map_at_1000_max': np.float64(0.8959021810971549),
   'nauc_map_at_1000_std': np.float64(0.8780411040820794),
   'nauc_map_at_1000_diff1': np.float64(1.0),
   'nauc_recall_at_1_max': np.float64(0.8921568627450974),
   'nauc_recall_at_1_std': np.float64(0.8552754435107388),
   'nauc_recall_at_1_diff1': np.float64(1.0),
   'nauc_recall_at_3_max': np.float64(0.8692810457516423),
   'nauc_recall_at_3_std': np.float64(1.0),
   'nauc_recall_at_3_diff1': np.float64(1.0),
   'nauc_recall_at_5_max': nan,
   'nauc_recall_at_5_std': nan,
   'nauc_recall_at_5_diff1': nan,
   'nauc_recall_at_10_max': nan,
   'nauc_recall_at_10_std': nan,
   'nauc_recall_at_10_diff1': nan,
   'nauc_recall_at_20_max': nan,
   'nauc_recall_at_20_std': nan,
   'nauc_recall_at_20_diff1': nan,
   'nauc_recall_at_100_max': nan,
   'nauc_recall_at_100_std': nan,
   'nauc_recall_at_100_diff1': nan,
   'nauc_recall_at_1000_max': nan,
   'nauc_recall_at_1000_std': nan,
   'nauc_recall_at_1000_diff1': nan,
   'nauc_precision_at_1_max': np.float64(0.8921568627450974),
   'nauc_precision_at_1_std': np.float64(0.8552754435107388),
   'nauc_precision_at_1_diff1': np.float64(1.0),
   'nauc_precision_at_3_max': np.float64(0.8692810457516407),
   'nauc_precision_at_3_std': np.float64(1.0),
   'nauc_precision_at_3_diff1': np.float64(1.0),
   'nauc_precision_at_5_max': np.float64(1.0),
   'nauc_precision_at_5_std': np.float64(1.0),
   'nauc_precision_at_5_diff1': np.float64(1.0),
   'nauc_precision_at_10_max': np.float64(1.0),
   'nauc_precision_at_10_std': np.float64(1.0),
   'nauc_precision_at_10_diff1': np.float64(1.0),
   'nauc_precision_at_20_max': np.float64(1.0),
   'nauc_precision_at_20_std': np.float64(1.0),
   'nauc_precision_at_20_diff1': np.float64(1.0),
   'nauc_precision_at_100_max': np.float64(1.0),
   'nauc_precision_at_100_std': np.float64(1.0),
   'nauc_precision_at_100_diff1': np.float64(1.0),
   'nauc_precision_at_1000_max': np.float64(1.0),
   'nauc_precision_at_1000_std': np.float64(1.0),
   'nauc_precision_at_1000_diff1': np.float64(1.0),
   'nauc_mrr_at_1_max': np.float64(0.8921568627450974),
   'nauc_mrr_at_1_std': np.float64(0.8552754435107388),
   'nauc_mrr_at_1_diff1': np.float64(1.0),
   'nauc_mrr_at_3_max': np.float64(0.8942208462332385),
   'nauc_mrr_at_3_std': np.float64(0.8857437711926923),
   'nauc_mrr_at_3_diff1': np.float64(1.0),
   'nauc_mrr_at_5_max': np.float64(0.8959021810971549),
   'nauc_mrr_at_5_std': np.float64(0.8780411040820794),
   'nauc_mrr_at_5_diff1': np.float64(1.0),
   'nauc_mrr_at_10_max': np.float64(0.8959021810971549),
   'nauc_mrr_at_10_std': np.float64(0.8780411040820794),
   'nauc_mrr_at_10_diff1': np.float64(1.0),
   'nauc_mrr_at_20_max': np.float64(0.8959021810971549),
   'nauc_mrr_at_20_std': np.float64(0.8780411040820794),
   'nauc_mrr_at_20_diff1': np.float64(1.0),
   'nauc_mrr_at_100_max': np.float64(0.8959021810971549),
   'nauc_mrr_at_100_std': np.float64(0.8780411040820794),
   'nauc_mrr_at_100_diff1': np.float64(1.0),
   'nauc_mrr_at_1000_max': np.float64(0.8959021810971549),
   'nauc_mrr_at_1000_std': np.float64(0.8780411040820794),
   'nauc_mrr_at_1000_diff1': np.float64(1.0),
   'hit_rate_at_1': 0.975,
   'hit_rate_at_3': 0.995,
   'hit_rate_at_5': 1.0,
   'hit_rate_at_10': 1.0,
   'hit_rate_at_20': 1.0,
   'hit_rate_at_100': 1.0,
   'hit_rate_at_1000': 1.0,
   'main_score': 0.9889,
   'hf_subset': 'pt',
   'languages': ['por-Latn'],
   'mteb_version': '2.19.5'}]}

# moBERTo
- main: 0.7377

{'test': [{'ndcg_at_1': 0.48,
   'ndcg_at_3': 0.64922,
   'ndcg_at_5': 0.70904,
   'ndcg_at_10': 0.7377,
   'ndcg_at_20': 0.7377,
   'ndcg_at_100': 0.7377,
   'ndcg_at_1000': 0.7377,
   'map_at_1': 0.48,
   'map_at_3': 0.6075,
   'map_at_5': 0.64075,
   'map_at_10': 0.65325,
   'map_at_20': 0.65325,
   'map_at_100': 0.65325,
   'map_at_1000': 0.65325,
   'recall_at_1': 0.48,
   'recall_at_3': 0.77,
   'recall_at_5': 0.915,
   'recall_at_10': 1.0,
   'recall_at_20': 1.0,
   'recall_at_100': 1.0,
   'recall_at_1000': 1.0,
   'accuracy': 0.48,
   'precision_at_1': 0.48,
   'precision_at_3': 0.25667,
   'precision_at_5': 0.183,
   'precision_at_10': 0.1,
   'precision_at_20': 0.05,
   'precision_at_100': 0.01,
   'precision_at_1000': 0.001,
   'mrr_at_1': 0.48,
   'mrr_at_3': 0.6075,
   'mrr_at_5': 0.64075,
   'mrr_at_10': 0.65325,
   'mrr_at_20': 0.65325,
   'mrr_at_100': 0.65325,
   'mrr_at_1000': 0.65325,
   'nauc_ndcg_at_1_max': np.float64(0.2826161744469145),
   'nauc_ndcg_at_1_std': np.float64(0.11577749550121726),
   'nauc_ndcg_at_1_diff1': np.float64(0.3998888536043192),
   'nauc_ndcg_at_3_max': np.float64(0.347837583405194),
   'nauc_ndcg_at_3_std': np.float64(0.15857509862483185),
   'nauc_ndcg_at_3_diff1': np.float64(0.3844663445847815),
   'nauc_ndcg_at_5_max': np.float64(0.34473012718245544),
   'nauc_ndcg_at_5_std': np.float64(0.11020792691214862),
   'nauc_ndcg_at_5_diff1': np.float64(0.3568568012871633),
   'nauc_ndcg_at_10_max': np.float64(0.31862295121484907),
   'nauc_ndcg_at_10_std': np.float64(0.12842902077734492),
   'nauc_ndcg_at_10_diff1': np.float64(0.3807455276249944),
   'nauc_ndcg_at_20_max': np.float64(0.31862295121484907),
   'nauc_ndcg_at_20_std': np.float64(0.12842902077734492),
   'nauc_ndcg_at_20_diff1': np.float64(0.3807455276249944),
   'nauc_ndcg_at_100_max': np.float64(0.31862295121484907),
   'nauc_ndcg_at_100_std': np.float64(0.12842902077734492),
   'nauc_ndcg_at_100_diff1': np.float64(0.3807455276249944),
   'nauc_ndcg_at_1000_max': np.float64(0.31862295121484907),
   'nauc_ndcg_at_1000_std': np.float64(0.12842902077734492),
   'nauc_ndcg_at_1000_diff1': np.float64(0.3807455276249944),
   'nauc_map_at_1_max': np.float64(0.2826161744469145),
   'nauc_map_at_1_std': np.float64(0.11577749550121726),
   'nauc_map_at_1_diff1': np.float64(0.3998888536043192),
   'nauc_map_at_3_max': np.float64(0.32781377202402084),
   'nauc_map_at_3_std': np.float64(0.14615260762013474),
   'nauc_map_at_3_diff1': np.float64(0.3853300492741933),
   'nauc_map_at_5_max': np.float64(0.3243961862753159),
   'nauc_map_at_5_std': np.float64(0.12101206670218684),
   'nauc_map_at_5_diff1': np.float64(0.372835002007239),
   'nauc_map_at_10_max': np.float64(0.3152081313573706),
   'nauc_map_at_10_std': np.float64(0.12812863719594217),
   'nauc_map_at_10_diff1': np.float64(0.38206236818977424),
   'nauc_map_at_20_max': np.float64(0.3152081313573706),
   'nauc_map_at_20_std': np.float64(0.12812863719594217),
   'nauc_map_at_20_diff1': np.float64(0.38206236818977424),
   'nauc_map_at_100_max': np.float64(0.3152081313573706),
   'nauc_map_at_100_std': np.float64(0.12812863719594217),
   'nauc_map_at_100_diff1': np.float64(0.38206236818977424),
   'nauc_map_at_1000_max': np.float64(0.3152081313573706),
   'nauc_map_at_1000_std': np.float64(0.12812863719594217),
   'nauc_map_at_1000_diff1': np.float64(0.38206236818977424),
   'nauc_recall_at_1_max': np.float64(0.2826161744469145),
   'nauc_recall_at_1_std': np.float64(0.11577749550121726),
   'nauc_recall_at_1_diff1': np.float64(0.3998888536043192),
   'nauc_recall_at_3_max': np.float64(0.4294330923658822),
   'nauc_recall_at_3_std': np.float64(0.20880239923009736),
   'nauc_recall_at_3_diff1': np.float64(0.3830711040487004),
   'nauc_recall_at_5_max': np.float64(0.5265282583621678),
   'nauc_recall_at_5_std': np.float64(-0.0035975174383479657),
   'nauc_recall_at_5_diff1': np.float64(0.20514087988136379),
   'nauc_recall_at_10_max': nan,
   'nauc_recall_at_10_std': nan,
   'nauc_recall_at_10_diff1': nan,
   'nauc_recall_at_20_max': nan,
   'nauc_recall_at_20_std': nan,
   'nauc_recall_at_20_diff1': nan,
   'nauc_recall_at_100_max': nan,
   'nauc_recall_at_100_std': nan,
   'nauc_recall_at_100_diff1': nan,
   'nauc_recall_at_1000_max': nan,
   'nauc_recall_at_1000_std': nan,
   'nauc_recall_at_1000_diff1': nan,
   'nauc_precision_at_1_max': np.float64(0.2826161744469145),
   'nauc_precision_at_1_std': np.float64(0.11577749550121726),
   'nauc_precision_at_1_diff1': np.float64(0.3998888536043192),
   'nauc_precision_at_3_max': np.float64(0.42943309236588195),
   'nauc_precision_at_3_std': np.float64(0.20880239923009725),
   'nauc_precision_at_3_diff1': np.float64(0.3830711040487002),
   'nauc_precision_at_5_max': np.float64(0.5265282583621683),
   'nauc_precision_at_5_std': np.float64(-0.0035975174383495066),
   'nauc_precision_at_5_diff1': np.float64(0.20514087988136517),
   'nauc_precision_at_10_max': np.float64(1.0),
   'nauc_precision_at_10_std': np.float64(1.0),
   'nauc_precision_at_10_diff1': np.float64(1.0),
   'nauc_precision_at_20_max': np.float64(1.0),
   'nauc_precision_at_20_std': np.float64(1.0),
   'nauc_precision_at_20_diff1': np.float64(1.0),
   'nauc_precision_at_100_max': np.float64(1.0),
   'nauc_precision_at_100_std': np.float64(1.0),
   'nauc_precision_at_100_diff1': np.float64(1.0),
   'nauc_precision_at_1000_max': np.float64(1.0),
   'nauc_precision_at_1000_std': np.float64(1.0),
   'nauc_precision_at_1000_diff1': np.float64(1.0),
   'nauc_mrr_at_1_max': np.float64(0.2826161744469145),
   'nauc_mrr_at_1_std': np.float64(0.11577749550121726),
   'nauc_mrr_at_1_diff1': np.float64(0.3998888536043192),
   'nauc_mrr_at_3_max': np.float64(0.32781377202402084),
   'nauc_mrr_at_3_std': np.float64(0.14615260762013474),
   'nauc_mrr_at_3_diff1': np.float64(0.3853300492741933),
   'nauc_mrr_at_5_max': np.float64(0.3243961862753159),
   'nauc_mrr_at_5_std': np.float64(0.12101206670218684),
   'nauc_mrr_at_5_diff1': np.float64(0.372835002007239),
   'nauc_mrr_at_10_max': np.float64(0.3152081313573706),
   'nauc_mrr_at_10_std': np.float64(0.12812863719594217),
   'nauc_mrr_at_10_diff1': np.float64(0.38206236818977424),
   'nauc_mrr_at_20_max': np.float64(0.3152081313573706),
   'nauc_mrr_at_20_std': np.float64(0.12812863719594217),
   'nauc_mrr_at_20_diff1': np.float64(0.38206236818977424),
   'nauc_mrr_at_100_max': np.float64(0.3152081313573706),
   'nauc_mrr_at_100_std': np.float64(0.12812863719594217),
   'nauc_mrr_at_100_diff1': np.float64(0.38206236818977424),
   'nauc_mrr_at_1000_max': np.float64(0.3152081313573706),
   'nauc_mrr_at_1000_std': np.float64(0.12812863719594217),
   'nauc_mrr_at_1000_diff1': np.float64(0.38206236818977424),
   'hit_rate_at_1': 0.48,
   'hit_rate_at_3': 0.77,
   'hit_rate_at_5': 0.915,
   'hit_rate_at_10': 1.0,
   'hit_rate_at_20': 1.0,
   'hit_rate_at_100': 1.0,
   'hit_rate_at_1000': 1.0,
   'main_score': 0.7377,
   'hf_subset': 'pt',
   'languages': ['por-Latn'],
   'mteb_version': '2.19.5'}]}

# BERTOMelo Base 8k
- Main: 0.9122

{'test': [{'ndcg_at_1': 0.81,
   'ndcg_at_3': 0.8894,
   'ndcg_at_5': 0.90382,
   'ndcg_at_10': 0.9122,
   'ndcg_at_20': 0.9122,
   'ndcg_at_100': 0.9122,
   'ndcg_at_1000': 0.9122,
   'map_at_1': 0.81,
   'map_at_3': 0.87167,
   'map_at_5': 0.87967,
   'map_at_10': 0.8833,
   'map_at_20': 0.8833,
   'map_at_100': 0.8833,
   'map_at_1000': 0.8833,
   'recall_at_1': 0.81,
   'recall_at_3': 0.94,
   'recall_at_5': 0.975,
   'recall_at_10': 1.0,
   'recall_at_20': 1.0,
   'recall_at_100': 1.0,
   'recall_at_1000': 1.0,
   'accuracy': 0.81,
   'precision_at_1': 0.81,
   'precision_at_3': 0.31333,
   'precision_at_5': 0.195,
   'precision_at_10': 0.1,
   'precision_at_20': 0.05,
   'precision_at_100': 0.01,
   'precision_at_1000': 0.001,
   'mrr_at_1': 0.81,
   'mrr_at_3': 0.8716666666666667,
   'mrr_at_5': 0.8796666666666667,
   'mrr_at_10': 0.883297619047619,
   'mrr_at_20': 0.883297619047619,
   'mrr_at_100': 0.883297619047619,
   'mrr_at_1000': 0.883297619047619,
   'nauc_ndcg_at_1_max': np.float64(0.5286945361202787),
   'nauc_ndcg_at_1_std': np.float64(0.35398658913510417),
   'nauc_ndcg_at_1_diff1': np.float64(0.6527057467651524),
   'nauc_ndcg_at_3_max': np.float64(0.5593985221146597),
   'nauc_ndcg_at_3_std': np.float64(0.4099298476783979),
   'nauc_ndcg_at_3_diff1': np.float64(0.6872017309897893),
   'nauc_ndcg_at_5_max': np.float64(0.5272370864951056),
   'nauc_ndcg_at_5_std': np.float64(0.38958830293351876),
   'nauc_ndcg_at_5_diff1': np.float64(0.6943710353283675),
   'nauc_ndcg_at_10_max': np.float64(0.5331062023956563),
   'nauc_ndcg_at_10_std': np.float64(0.38522784060025683),
   'nauc_ndcg_at_10_diff1': np.float64(0.6731845452157957),
   'nauc_ndcg_at_20_max': np.float64(0.5331062023956563),
   'nauc_ndcg_at_20_std': np.float64(0.38522784060025683),
   'nauc_ndcg_at_20_diff1': np.float64(0.6731845452157957),
   'nauc_ndcg_at_100_max': np.float64(0.5331062023956563),
   'nauc_ndcg_at_100_std': np.float64(0.38522784060025683),
   'nauc_ndcg_at_100_diff1': np.float64(0.6731845452157957),
   'nauc_ndcg_at_1000_max': np.float64(0.5331062023956563),
   'nauc_ndcg_at_1000_std': np.float64(0.38522784060025683),
   'nauc_ndcg_at_1000_diff1': np.float64(0.6731845452157957),
   'nauc_map_at_1_max': np.float64(0.5286945361202787),
   'nauc_map_at_1_std': np.float64(0.35398658913510417),
   'nauc_map_at_1_diff1': np.float64(0.6527057467651524),
   'nauc_map_at_3_max': np.float64(0.5461321438102869),
   'nauc_map_at_3_std': np.float64(0.3938325263385314),
   'nauc_map_at_3_diff1': np.float64(0.6746127060378467),
   'nauc_map_at_5_max': np.float64(0.5302452743989489),
   'nauc_map_at_5_std': np.float64(0.3833614109226586),
   'nauc_map_at_5_diff1': np.float64(0.6779388330056483),
   'nauc_map_at_10_max': np.float64(0.5326404818224778),
   'nauc_map_at_10_std': np.float64(0.38269496629475414),
   'nauc_map_at_10_diff1': np.float64(0.6707393107573829),
   'nauc_map_at_20_max': np.float64(0.5326404818224778),
   'nauc_map_at_20_std': np.float64(0.38269496629475414),
   'nauc_map_at_20_diff1': np.float64(0.6707393107573829),
   'nauc_map_at_100_max': np.float64(0.5326404818224778),
   'nauc_map_at_100_std': np.float64(0.38269496629475414),
   'nauc_map_at_100_diff1': np.float64(0.6707393107573829),
   'nauc_map_at_1000_max': np.float64(0.5326404818224778),
   'nauc_map_at_1000_std': np.float64(0.38269496629475414),
   'nauc_map_at_1000_diff1': np.float64(0.6707393107573829),
   'nauc_recall_at_1_max': np.float64(0.5286945361202787),
   'nauc_recall_at_1_std': np.float64(0.35398658913510417),
   'nauc_recall_at_1_diff1': np.float64(0.6527057467651524),
   'nauc_recall_at_3_max': np.float64(0.6396280734516037),
   'nauc_recall_at_3_std': np.float64(0.5017507002801125),
   'nauc_recall_at_3_diff1': np.float64(0.7617491441020879),
   'nauc_recall_at_5_max': np.float64(0.47721755368814245),
   'nauc_recall_at_5_std': np.float64(0.45434173669468536),
   'nauc_recall_at_5_diff1': np.float64(0.9108309990662924),
   'nauc_recall_at_10_max': nan,
   'nauc_recall_at_10_std': nan,
   'nauc_recall_at_10_diff1': nan,
   'nauc_recall_at_20_max': nan,
   'nauc_recall_at_20_std': nan,
   'nauc_recall_at_20_diff1': nan,
   'nauc_recall_at_100_max': nan,
   'nauc_recall_at_100_std': nan,
   'nauc_recall_at_100_diff1': nan,
   'nauc_recall_at_1000_max': nan,
   'nauc_recall_at_1000_std': nan,
   'nauc_recall_at_1000_diff1': nan,
   'nauc_precision_at_1_max': np.float64(0.5286945361202787),
   'nauc_precision_at_1_std': np.float64(0.35398658913510417),
   'nauc_precision_at_1_diff1': np.float64(0.6527057467651524),
   'nauc_precision_at_3_max': np.float64(0.6396280734515983),
   'nauc_precision_at_3_std': np.float64(0.5017507002801097),
   'nauc_precision_at_3_diff1': np.float64(0.7617491441020812),
   'nauc_precision_at_5_max': np.float64(0.47721755368813745),
   'nauc_precision_at_5_std': np.float64(0.4543417366946735),
   'nauc_precision_at_5_diff1': np.float64(0.9108309990662947),
   'nauc_precision_at_10_max': np.float64(1.0),
   'nauc_precision_at_10_std': np.float64(1.0),
   'nauc_precision_at_10_diff1': np.float64(1.0),
   'nauc_precision_at_20_max': np.float64(1.0),
   'nauc_precision_at_20_std': np.float64(1.0),
   'nauc_precision_at_20_diff1': np.float64(1.0),
   'nauc_precision_at_100_max': np.float64(1.0),
   'nauc_precision_at_100_std': np.float64(1.0),
   'nauc_precision_at_100_diff1': np.float64(1.0),
   'nauc_precision_at_1000_max': np.float64(1.0),
   'nauc_precision_at_1000_std': np.float64(1.0),
   'nauc_precision_at_1000_diff1': np.float64(1.0),
   'nauc_mrr_at_1_max': np.float64(0.5286945361202787),
   'nauc_mrr_at_1_std': np.float64(0.35398658913510417),
   'nauc_mrr_at_1_diff1': np.float64(0.6527057467651524),
   'nauc_mrr_at_3_max': np.float64(0.5461321438102869),
   'nauc_mrr_at_3_std': np.float64(0.3938325263385314),
   'nauc_mrr_at_3_diff1': np.float64(0.6746127060378467),
   'nauc_mrr_at_5_max': np.float64(0.5302452743989489),
   'nauc_mrr_at_5_std': np.float64(0.3833614109226586),
   'nauc_mrr_at_5_diff1': np.float64(0.6779388330056483),
   'nauc_mrr_at_10_max': np.float64(0.5326404818224778),
   'nauc_mrr_at_10_std': np.float64(0.38269496629475414),
   'nauc_mrr_at_10_diff1': np.float64(0.6707393107573829),
   'nauc_mrr_at_20_max': np.float64(0.5326404818224778),
   'nauc_mrr_at_20_std': np.float64(0.38269496629475414),
   'nauc_mrr_at_20_diff1': np.float64(0.6707393107573829),
   'nauc_mrr_at_100_max': np.float64(0.5326404818224778),
   'nauc_mrr_at_100_std': np.float64(0.38269496629475414),
   'nauc_mrr_at_100_diff1': np.float64(0.6707393107573829),
   'nauc_mrr_at_1000_max': np.float64(0.5326404818224778),
   'nauc_mrr_at_1000_std': np.float64(0.38269496629475414),
   'nauc_mrr_at_1000_diff1': np.float64(0.6707393107573829),
   'hit_rate_at_1': 0.81,
   'hit_rate_at_3': 0.94,
   'hit_rate_at_5': 0.975,
   'hit_rate_at_10': 1.0,
   'hit_rate_at_20': 1.0,
   'hit_rate_at_100': 1.0,
   'hit_rate_at_1000': 1.0,
   'main_score': 0.9122,
   'hf_subset': 'pt',
   'languages': ['por-Latn'],
   'mteb_version': '2.19.5'}]}
